src/Nethermind/Nethermind.Core.Test/FixedBlockChainHeadSpecProvider.cs

```diff
@@ -48,6 +48,6 @@ public FixedBlockChainHeadSpecProvider(ISpecProvider specProvider, long fixedBlo
 
         public ForkActivation[] TransitionBlocks => _specProvider.TransitionBlocks;
 
-        public IReleaseSpec GetCurrentHeadSpec() => GetSpec(_fixedBlock);
+        public IReleaseSpec GetCurrentHeadSpec() => GetSpec((ForkActivation)_fixedBlock);
```

Comment:
Should we changed this to FixedForkActivationChainHeadSpecProvider and provide optional Timestamp?